compiler/control/OMROptions.cpp

```diff
@@ -3201,6 +3201,11 @@ const char *OMR::Options::processOptionsAOT(const char *aotOptions, void *feBase
     else
         _aotCmdLineOptions->init();
 
+    if (!_jitCmdLineOptions) {
+        _jitCmdLineOptions = new (PERSISTENT_NEW) TR::Options();
+        _cmdLineOptions = _jitCmdLineOptions;
+    }
+
```

Comment:
I think you'll need an else in case `_jitCmdLineOptions` is non NULL but `_cmdLineOptions` is uninitialized (somehow..). 

Maybe it's better to:
```suggestion
    if (!_jitCmdLineOptions) {
        _jitCmdLineOptions = new (PERSISTENT_NEW) TR::Options();
        _cmdLineOptions = _jitCmdLineOptions;
    } else {
        TR_ASSERT_FATAL(_jitCmdLineOptions == _cmdLineOptions, "...");
    }
```

Also, as Marius said offline, you'll likely need to do `_jitCmdLineOptions->init()`, though that will result in `init()` getting called more than once, so you'll have to verify that as well.